packages/repository-tests/src/crud/relations/fixtures/models/customer.model.ts

```diff
@@ -1,19 +1,31 @@
 // Copyright IBM Corp. 2019. All Rights Reserved.
-// Node module: @loopback/repository
+// Node module: @loopback/repository-tests
 // This file is licensed under the MIT License.
 // License text available at https://opensource.org/licenses/MIT
 
-import {belongsTo, Entity, hasMany, hasOne, model, property} from '../../..';
+import {
+  belongsTo,
+  Entity,
+  hasMany,
+  hasOne,
+  model,
+  property,
+} from '@loopback/repository';
 import {Address, AddressWithRelations} from './address.model';
 import {Order, OrderWithRelations} from './order.model';
 
-@model()
+@model({
+  settings: {
+    strictObjectIDCoercion: true,
+  },
+})
 export class Customer extends Entity {
   @property({
-    type: 'number',
+    type: 'string',
     id: true,
+    generated: true,
```

Comment:
It makes me wonder - why is it necessary to set `generated` to `true`? Isn't it implied automatically for `id` properties? Is this needed to workaround ObjectID issues in MongoDB? 

Anyhow, if we need to enable `generated: true` to make relations work, then again this needs to be clearly documented in our docs and ideally `lb4 model` should set that flag automatically. Such change is out of scope of this pull request, as I mentioned earlier. I think we can open just one issue to make all changes needed to make relations work in MongoDB.

Reply:
Makes sense :+1:

Does `lb4 model` set `generated: true` too? If not, then can you open a new GH issue to make that change please? Or even open a new PR, it may be faster than going through issue triage+estimation+planning process.

Reply:
> Does `lb4 model` set `generated: true` too? If not, then can you open a new GH issue to make that change please? Or even open a new PR, it may be faster than going through issue triage+estimation+planning process.

☝️ PTAL

Reply:
@bajtos https://github.com/strongloop/loopback-next/pull/3643